### PR TITLE
updated modal docs of custom auth

### PR DESCRIPTION
**Pull request by Saviour1001**

### CODE REVIEW

File: docs/sdk/web/modal/custom-authentication.mdx [hunk 1]
```diff
@@ -82,4 +82,97 @@ The `loginConfig` parameter of `adapterSettings` in `openlogin-adapter` contains
...
+
+const openloginAdapter = new OpenloginAdapter({
+  adapterSettings: {
+    clientId, //Optional - Provide only if you haven't provided it in the Web3Auth Instantiation Code
+    network: "testnet",
+    uxMode: "popup",
+    // highlight-start
+    loginConfig: {
+      // JWT login
+      jwt: {
```
Comment (shahbaz17) [inline docs/sdk/web/modal/custom-authentication.mdx:163]: This is wrong, there is no JWT with Modal.

**Review by yashovardhan — CHANGES_REQUESTED** [review]
Just update the common file for `CustomAuthenticationWeb3AuthExample` to use this.

**Comment by Saviour1001** [comment]
Done

**Review by yashovardhan — APPROVED** [review]
LGTM

**Pull request merged**